app/models/katello/events/import_host_applicability.rb

```diff
@@ -5,7 +5,7 @@ class ImportHostApplicability
 
       def initialize(object_id)
         @host = ::Host.find_by_id(object_id)
-        fail "Host not found for ID #{object_id}" if @host.nil?
+        Rails.logger.warn "Host not found for ID #{object_id}" if @host.nil?
```

Comment:
@jlsherrill if its ok with you, lets just go with this patch as-is since i already tested it.